### PR TITLE
added step by step instruction on adding an image to a website

### LINKED ISSUE #13261: Add Images to your Website - Clarification needed in instruction
Challenge [Add Images to your Website](https://www.freecodecamp.com/challenges/add-images-to-your-website) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

*This challenge does not give instructions on if the user should put a img tag and where it should be positioned. My reason for this issue is because in the previous challenges it would give a specific instruction on what to do.

My code:
```html

<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
<style>
  .red-text {
    color: red;
  }

  h2 {
    font-family: Lobster, Monospace;
  }

  p {
    font-size: 16px;
    font-family: Monospace;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
<p class="red-text">Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>

```

### PR DESCRIPTION
**Pull request by Marmiz**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13261

#### Description

As per #13261 I have added some step by step instruction to ease the understanding of the image tag:

>Let's try to add an image to our website:
        Before the <code>h2</code> element insert an <code>img</code> tag.
        Now set the <code>src</code> attribute so that it point to this url:
        <code>https://bit.ly/fcc-relaxing-cat</code>
        Finally don't forget to give your image an <code>alt</code> text.

### CODE REVIEW

**Comment by Marmiz** [comment]
Just noticed a small typo:

I have written:

> Now set the src attribute so that it point to this url:

Should probably be:

> Now set the src attribute so that it **points** to this url:

Can I change and commit without re-submitting a PR?

**Comment by raisedadead** [comment]
> Can I change and commit without re-submitting a PR?

Yes, just to the below:
1. `git checkout  fix/Add-images-challenge `
2. Edit the file to make the necessary changes 
3. `git add --all`
4. `git commit --amend --no-edit`
5. `git push origin --force`

This will update the PR automatically.

**Comment by raisedadead** [comment]
Also please edit the PR description in the Opening Post above to modify the closes tag refering the issue, this will automatically close the open issue.

**Comment by Marmiz** [comment]
@raisedadead followed the instruction 😄 
I've edited the PR template (hopefully correctly); I didn't know that will close the issue automatically.

I've learned another new thing today!
Thanks for your help 💙

File: seed/challenges/01-responsive-web-design/basic-html-and-html5.json [hunk 1]
```diff
@@ -570,8 +570,11 @@
         "<code>&#60img src=\"https://www.your-image-source.com/your-image.jpg\" alt=\"Author standing on a beach with two thumbs up.\"&#62</code>",
         "Note that in most cases, <code>img</code> elements are self-closing.",
         "<hr>",
-        "Try it with this image:",
-        "<code>https://bit.ly/fcc-relaxing-cat</code>"
+        "Let's try to add an image to our website:",
+        "Before the <code>h2</code> element insert an <code>img</code> tag.",
```
Comment (raisedadead) [inline seed/challenges/01-responsive-web-design/basic-html-and-html5.json]: Update the statement:
```diff
- "Before the <code>h2</code> element insert an <code>img</code> tag.",
+ "Insert an <code>img</code> tag, before the <code>h2</code> element.",
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Just a minor change and we are good to go.

**Comment by Marmiz** [comment]
@raisedadead pushed the change as requested:
> "Insert an <code>img</code> tag, before the <code>h2</code> element.",

Apparently now there's a vulnerable path, the "details" show is linked to the `marked` packages that I haven't touched.
Suggestion on a fix?

I haven't done a `git pull` before re-committing the change, perhaps was that?

**Comment by teeleek** [comment]
Okay

Puttasak insee

เมื่อ 13 ก.พ. 2017 09:46 "Claudio Restifo" <notifications@github.com>
เขียนว่า

**Comment by raisedadead** [comment]
> Apparently now there's a vulnerable path, the "details" show is linked to the marked packages that I haven't touched.

Yes, this doesn't relate to your PR, a re-base against the `staging` branch will fix this for your branch.

This happened, when we patched a pre-existinf vulnerability, and your PR had been open before that, a simple re-base would take care of this.

```
git checkout <branch name>
git pull --rebase upstream staging
git push origin --force
```

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Changes LGTM. Ready for a merge pending re-base.

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @Marmiz for your contribution!

**Pull request merged**